azure_arc_servers_jumpstart/docs/azure_arm_linux.md

```diff
@@ -0,0 +1,86 @@
+#  Onboard an Azure Linux Server VM with Azure Arc
+
+The following README will guide you on how onboard a Azure Linux VM on to Azure Arc. An ARM template is provided for the creation of the Azure Resouces, along with an script that will allow you to onboard the Azure VM onto Azure Arc, this step is requiered as Azure VMs are already part of ARM, therefore, the Azure Arc agent cannot be installed following the regular onboarding method. 
+
+   > [!NOTE]Please note that this scenario is only intended for demo purposes. 
+
+# Azure Account  
+
+* You will need an Azure Account with an active and valid subscription so you can deploy the Azure VMs and then register and onboard them with Azure Arc. If you do not have an account already, you can start with a free-trial account. 
+
+* To create an Azure free account browse to [this link](https://azure.microsoft.com/en-us/free/) and select 'Start Free' to get access to a free trial subscription. 
+
+# Prerequisites
+
+* Clone this repo
+
+    ```terminal
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+    
+* [Install or update Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). **Azure CLI should be running version 2.7** or later. Use ```az --version``` to check your current installed version.
+
+# Automation Flow
```

Comment:
Please align to the language [here](https://github.com/microsoft/azure_arc/blob/master/azure_arc_sqlsrv_jumpstart/docs/aws_terraform_winsrv.md#automation-flow)